serverHandlers/middleware.js

```diff
@@ -21,36 +21,36 @@ exports.validateInputFields = async (req, res, next) => {
         fs.unlinkSync(path.join(__dirname, '..', imgPath));
     };
 
-    UserModel.find({userEmail: checkEmail}, (err, data) => {
+    UserModel.find({email: checkEmail}, (err, data) => {
         if(data.length){
             imgRemove(imgPath);
             res.status(400).end(JSON.stringify({message: "User already exists!"}));
+        } else {
+            // validation check
+            if(!(/^([a-zA-Z]{1,32})$/.test(userInfo.name))){
```

Comment:
In all this if..else if statements as far as I see - different are only messages. Other commands are equal.
Would be awesome if you will do the next code after the checks:
```
imgRemove(imgPath);
res.status(400).json({message: myMessage});
```

Reply:
optimized server response